app/javascript/components/admin/AdminNavSideBar.jsx

```diff
@@ -32,18 +34,22 @@ export default function AdminNavSideBar() {
           </Nav.Link>
         </Nav.Item>
       )}
-      <Nav.Item>
-        <Nav.Link className="cursor-pointer text-muted" as={Link} to="/adminpanel/site-settings" eventKey="site-settings">
-          <CogIcon className="hi-s me-3" />
-          Site Settings
-        </Nav.Link>
-      </Nav.Item>
-      <Nav.Item>
-        <Nav.Link className="cursor-pointer text-muted" as={Link} to="/adminpanel/room-configuration" eventKey="room-configuration">
-          <AdjustmentsIcon className="hi-s me-3" />
-          Room Configuration
-        </Nav.Link>
-      </Nav.Item>
+      {(currentUser.permissions.ManageSiteSettings === 'true') && (
+        <Nav.Item>
+          <Nav.Link className="cursor-pointer text-muted" as={Link} to="/adminpanel/site-settings" eventKey="site-settings">
+            <CogIcon className="hi-s me-3" />
+            Site Settings
+          </Nav.Link>
+        </Nav.Item>
+      )}
+      {(currentUser.permissions.ManageSiteSettings === 'true') && (
+        <Nav.Item>
+          <Nav.Link className="cursor-pointer text-muted" as={Link} to="/adminpanel/room-configuration" eventKey="room-configuration">
+            <AdjustmentsIcon className="hi-s me-3" />
+            Room Configuration
+          </Nav.Link>
+        </Nav.Item>
+      )}
```

Comment:
You can just do this in 1 block - no need to check twice

Reply:
Is there a specific permission for Room Configs? Is it SiteSettings?

Reply:
Yeah